src/SFA.DAS.EmployerIncentives.Api/Authorization/AuthorizationPolicyBuilderExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Authorization;
+
+namespace SFA.DAS.EmployerIncentives.Api.Authorization
+{
+    public static class AuthorizationPolicyBuilderExtensions
```

Comment:
As per previous, looks like this could be shared?

Reply:
agreed